llvm/lib/SYCLLowerIR/CMakeLists.txt

```diff
@@ -12,11 +12,8 @@ endif()
 # NOTE: could have been added earlier from llvm/projects
 if (NOT TARGET LLVMGenXIntrinsics)
   if (NOT DEFINED LLVMGenXIntrinsics_SOURCE_DIR)
-    set(LLVMGenXIntrinsics_GIT_REPO https://github.com/intel/vc-intrinsics.git)
-    # Author: Gorban, Igor <igor.gorban@intel.com>
-    # Date:   Tue Jun 6 23:56:43 2023 +0000
-    # Mark llvm15 tests failures
-    set(LLVMGenXIntrinsics_GIT_TAG 910db4801d4a029834606e3e42a8d60358e74fdf)
+    set(LLVMGenXIntrinsics_GIT_REPO https://github.com/jsji/vc-intrinsics.git)
```

Comment:
I see the corresponding PR has been merged. Think this is ready for update. Just a reminder. Thanks

Reply:
Yes, but I am still waiting for the sync to open source repo. https://github.com/intel/vc-intrinsics

Reply:
Finally updated!